### PR TITLE
Updated Mac install steps

### PR DESCRIPTION
**Pull request by MechMel**
I updated the Mac build steps to try to counter some issues I ran into. I have an M2 Mac.

### CODE REVIEW

**Comment by laanwj** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -80,16 +80,20 @@ ninja -C build
    ```
 5. Install the built binary somewhere *not* in the `PATH` (see below), e.g.
    ```
-   mkdir /usr/local/lib/sundog
+   sudo mkdir /usr/local/lib/sundog
    cp build/src/sundog /usr/local/lib/sundog/
```
Comment (laanwj) [inline README.md:84]: Doesn't this copy step need sudo privileges as well?

Comment (MechMel) [reply README.md:84]: It probably should. Sometimes one "sudo" was all it wanted, and sometimes it wanted me to enter it twice. I'm not exactly sure where this rule is applied. I'm new to Macs.

Comment (laanwj) [reply README.md:84]: Right, the reasoning would be that `sudo mkdir` creates a directory with root ownership and not user ownership, so copying a file there requires root privs as well. I don't expect MacOS to be very different than other UNIX there.

**Pull request closed without merging**